docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: "Migrating from Thanos or Prometheus to Grafana Mimir"
-menuTitle: "Migrating from Thanos or Prometheus"
+aliases:
+  - migrating-from-thanos-or-prometheus/
```

Comment:
@jdbaldry, is this alias okay?

Reply:
I haven't manually checked it, but it's the same pattern we used for many other aliases where we just renamed the title.

Reply:
You don't need an alias in this case because the page hasn't moved.

Reply:
An alias is a redirect from one page to another. We have some docs in https://grafana.com/docs/writers-toolkit/writing-guide/front-matter/#hugo-aliases.

Reply:
On right, the filename hasn't changed, but maybe Ursula meant to also change the filename? 🤔

Reply:
Probably, the PR title suggests that was intended.

Assuming the rename, the way I would check is to run the local webserver and browse to the old page location (http://localhost:3002/docs/mimir/latest/migrate/migrating-from-thanos-or-prometheus/). If the alias is correct, you should be redirected to the new page location.

Reply:
I wrote https://github.com/grafana/technical-documentation/blob/main/tools/cmd/link-checker/main.go to do this for me but I must admit I haven't run it in a while :D

Reply:
I think some assumptions about directory structure means that we can't yet use it here (https://github.com/grafana/technical-documentation/blob/main/tools/cmd/link-checker/main.go#L24, for example)

Reply:
> On right, the filename hasn't changed, but maybe Ursula meant to also change the filename? thinking

Yes, indeed, I meant to do that. heh

Reply:
Classic result of multi-tasking. Thank you for catching that!